<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vampConfigDialog</class>
 <widget class="QDialog" name="vampConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project Configuration Options</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="configTabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="instTab">
      <attribute name="title">
       <string>Instrumentation Config</string>
      </attribute>
      <widget class="QLabel" name="mcdcStackSizeLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>MC/DC Stack Size</string>
       </property>
       <property name="buddy">
        <cstring>mcdcStackSize</cstring>
       </property>
      </widget>
      <widget class="QLineEdit" name="mcdcStackSize">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>200</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="inputMask">
        <string>####</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>196</width>
         <height>29</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="saveSuffixLabel">
          <property name="text">
           <string>Save Suffix     </string>
          </property>
          <property name="buddy">
           <cstring>saveSuffix</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="saveSuffix"/>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="langStdGroupBox">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>239</y>
         <width>194</width>
         <height>142</height>
        </rect>
       </property>
       <property name="title">
        <string>Language Standard</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QRadioButton" name="C1990_RadioButton">
          <property name="text">
           <string>ISO C 1990</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="C1999_RadioButton">
          <property name="text">
           <string>ISO C 1999</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="C2011_RadioButton">
          <property name="text">
           <string>ISO C 2011</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="gnuExt_checkBox">
          <property name="text">
           <string>Has GNU Extensions</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="saveDirButton">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>130</y>
         <width>31</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>101</y>
         <width>441</width>
         <height>52</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="saveDirectoryLabel">
          <property name="text">
           <string>Save Directory</string>
          </property>
          <property name="buddy">
           <cstring>saveDirectory</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="saveDirectory"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>20</y>
         <width>474</width>
         <height>80</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QCheckBox" name="singleStmtCoverage">
          <property name="text">
           <string>Single Statement Coverage</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>28</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2">
         <widget class="QCheckBox" name="countStmtCoverage">
          <property name="text">
           <string>Count Statement Coverage</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="mcdcCoverage">
          <property name="text">
           <string>MC/DC Coverage</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>28</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="2">
         <widget class="QCheckBox" name="conditionCoverage">
          <property name="text">
           <string>Basic Condition Coverage</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="branchCoverage">
          <property name="text">
           <string>Branch Coverage</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="reportTab">
      <attribute name="title">
       <string>Report/History Config</string>
      </attribute>
      <widget class="QGroupBox" name="reportGroupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>260</y>
         <width>481</width>
         <height>211</height>
        </rect>
       </property>
       <property name="title">
        <string>Report</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <widget class="QPushButton" name="htmlDirButton">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>130</y>
          <width>31</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="showTestCases">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>251</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Show Recommended Test Cases</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="reportSeparator">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>60</y>
          <width>20</width>
          <height>20</height>
         </rect>
        </property>
        <property name="maxLength">
         <number>1</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>60</y>
          <width>191</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Report Separator Character</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="generateReport">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>60</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Generate RPT Report</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>8</x>
          <y>160</y>
          <width>421</width>
          <height>29</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>HTML File Suffix</string>
           </property>
           <property name="buddy">
            <cstring>htmlSuffix</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="htmlSuffix"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>100</y>
          <width>421</width>
          <height>52</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>HTML Save Directory</string>
           </property>
           <property name="buddy">
            <cstring>htmlDirectory</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLineEdit" name="htmlDirectory"/>
         </item>
        </layout>
       </widget>
       <zorder>layoutWidget</zorder>
       <zorder>layoutWidget</zorder>
       <zorder>htmlDirButton</zorder>
       <zorder>showTestCases</zorder>
       <zorder>reportSeparator</zorder>
       <zorder>label_4</zorder>
       <zorder>generateReport</zorder>
      </widget>
      <widget class="QGroupBox" name="historyGroupBox">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>9</y>
         <width>481</width>
         <height>181</height>
        </rect>
       </property>
       <property name="title">
        <string>History</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <widget class="QCheckBox" name="combineHistory">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>80</y>
          <width>361</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Combine Histories</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>361</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>History Directory</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="histDirectory">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>50</y>
          <width>431</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="histDirButton">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>50</y>
          <width>31</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="preprocessorTab">
      <attribute name="title">
       <string>Preprocessor Config</string>
      </attribute>
      <widget class="QPushButton" name="preProcDirButton">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>170</y>
         <width>31</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="preProcCheckBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Preprocess Files</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>441</width>
         <height>62</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="pathToPreProcLabel">
          <property name="text">
           <string>Path to Preprocessor</string>
          </property>
          <property name="buddy">
           <cstring>preProcPathEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="preProcPathEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="preProcArgsLabel">
          <property name="text">
           <string>Preprocessor Arguments</string>
          </property>
          <property name="buddy">
           <cstring>preProcArgsEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="preProcArgsEdit"/>
        </item>
       </layout>
      </widget>
      <widget class="QTabWidget" name="preprocOptionsTabWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>481</width>
         <height>231</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="Includes">
        <attribute name="title">
         <string>Include Dirs</string>
        </attribute>
        <widget class="QGroupBox" name="groupBox">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>481</width>
           <height>211</height>
          </rect>
         </property>
         <property name="title">
          <string>Include Directories</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <widget class="QPlainTextEdit" name="addIncludeTextEdit">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>431</width>
            <height>181</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="addIncludeButton">
          <property name="geometry">
           <rect>
            <x>450</x>
            <y>100</y>
            <width>31</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="Defines">
        <attribute name="title">
         <string>Defines</string>
        </attribute>
        <widget class="QTextEdit" name="definesTextEdit">
         <property name="geometry">
          <rect>
           <x>3</x>
           <y>10</y>
           <width>471</width>
           <height>181</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QCheckBox" name="useVampPreprocCheckBox">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>10</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Use VAMP preprocessor</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="retainCommentsCheckBox">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>30</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Retain Comments</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="buildPPMapCheckBox">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>50</y>
         <width>261</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Build Preprocessed to Source Map</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>141</y>
         <width>441</width>
         <height>52</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="preProcDirLabel">
          <property name="text">
           <string>Preprocess Save Directory</string>
          </property>
          <property name="buddy">
           <cstring>preProcDirectory</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="preProcDirectory"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>201</y>
         <width>441</width>
         <height>29</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="preProcSuffixLabel">
          <property name="text">
           <string>Preprocess Suffix</string>
          </property>
          <property name="buddy">
           <cstring>preProcSuffix</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="preProcSuffix"/>
        </item>
       </layout>
      </widget>
      <zorder>layoutWidget</zorder>
      <zorder>layoutWidget</zorder>
      <zorder>layoutWidget</zorder>
      <zorder>preProcDirButton</zorder>
      <zorder>preProcCheckBox</zorder>
      <zorder>preprocOptionsTabWidget</zorder>
      <zorder>useVampPreprocCheckBox</zorder>
      <zorder>retainCommentsCheckBox</zorder>
      <zorder>buildPPMapCheckBox</zorder>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>singleStmtCoverage</tabstop>
  <tabstop>countStmtCoverage</tabstop>
  <tabstop>mcdcCoverage</tabstop>
  <tabstop>conditionCoverage</tabstop>
  <tabstop>branchCoverage</tabstop>
  <tabstop>saveDirectory</tabstop>
  <tabstop>saveSuffix</tabstop>
  <tabstop>mcdcStackSize</tabstop>
  <tabstop>C1990_RadioButton</tabstop>
  <tabstop>C1999_RadioButton</tabstop>
  <tabstop>C2011_RadioButton</tabstop>
  <tabstop>gnuExt_checkBox</tabstop>
  <tabstop>combineHistory</tabstop>
  <tabstop>showTestCases</tabstop>
  <tabstop>generateReport</tabstop>
  <tabstop>reportSeparator</tabstop>
  <tabstop>htmlDirButton</tabstop>
  <tabstop>preProcCheckBox</tabstop>
  <tabstop>useVampPreprocCheckBox</tabstop>
  <tabstop>retainCommentsCheckBox</tabstop>
  <tabstop>buildPPMapCheckBox</tabstop>
  <tabstop>preProcPathEdit</tabstop>
  <tabstop>preProcArgsEdit</tabstop>
  <tabstop>preProcDirectory</tabstop>
  <tabstop>preProcDirButton</tabstop>
  <tabstop>preProcSuffix</tabstop>
  <tabstop>preprocOptionsTabWidget</tabstop>
  <tabstop>addIncludeTextEdit</tabstop>
  <tabstop>addIncludeButton</tabstop>
  <tabstop>definesTextEdit</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>configTabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>vampConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>521</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>vampConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>521</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
